### PR TITLE
Merge new Python ICIDS2017 case-study software into main repository

### PR DESCRIPTION
**Pull request by WillBridges1999**
Changes proposed in this pull request:

- Add new Python case-study software for the ICIDS2017 dataset/ case-study (from BSc dissertation work).

- New additions to the **implementation_in_Python/CICIDS2017_Security_Data** folder only.

### CODE REVIEW

File: Implementation_in_Python/CICIDS2017_Security_Data/Sample_Results_Security_Example/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ <figcaption>Figure 1. Confusion matrix for Wednesday Security Intrusion Detection in CICIDS2017 Dataset</figcaption>
+</p>
+<p align="justify">Figure 2 shows a sample result of six statistical measures (one pdf-based and five cdf-based) vs. accuracy of the classifier. In this example, traditional classifiers including 'Naive Bayes','Discriminant Analysis','Classification Tree', and 'Nearest Neighbor' have been used. </p>
+<p align="center">
+ <img src="https://github.com/ISorokos/SafeML/blob/master/Implementation_in_MATLAB/Sample_Results_Security_Example/Security_Dataset_W02v2.png" alt="Wednessday_Attack">
+ <figcaption>Figure 2. Sample statistical distance measures vs. accuracy</figcaption>
+</p>
+<p align="justify">The above figure has been generated over 100 times, and each time 70% of data has been randomly extracted for testing and training and the rest has been used for accuracy estimation. If we connect red lines of each box plot we can understand how each measure can react to the accuracy change on average. In addition, this plot can show which method has less variation.</p>
```
Comment (ISorokos) [inline Implementation_in_Python/CICIDS2017_Security_Data/Sample_Results_Security_Example/README.md:13]: @WillBridges1999 "the above figure has been generated over 100 times", do you mean the data/experiments were executed over (more than?) 100 iterations ?

**Review by ISorokos — APPROVED** [review]
Very well put together ! 
Thank you very much for the contribution and sorry for the slow response :)
I've left a clarification comment, but the rest is fine by me.

File: Implementation_in_Python/CICIDS2017_Security_Data/Sample_Results_Security_Example/README.md (hunk 1 above)
Comment (WillBridges1999) [reply Implementation_in_Python/CICIDS2017_Security_Data/Sample_Results_Security_Example/README.md:13]: Sorry, I think that's just the copied README from the MATLAB code implementation. I should probably have taken that out but it's up to you :) That part you've mentioned seems like a mistake for sure. Good spot!

**Comment by WillBridges1999** [comment]
Thank you! I'm glad you're happy with it! @ISorokos 
No worries Dr Sorokos, and it's been my pleasure- @koo-ec guided me extremely well and helped me continuously, so I'm very grateful for all his help.
Thanks for reviewing and merging my work; hopefully I can continue to work with you on this project, after I've secured a graduate role in the industry.

**Pull request merged**